### DISCUSSION TITLE
Upload file

### DISCUSSION THREAD

**Discussion by Leoskari** [kimai/kimai#4422] [General] [answered]
Hi, is there way to upload file (employment contract - absense - sickness) like doctor report of sickness? Or any other way?

**Comment by kevinpapst** [comment]
No.

**Reply by kevinpapst** [reply, marked as answer]
Put it in your Google Drive / Dropbox / iCloud Drive in a separate folder per user.
Create a custom URL field for User.
Add the link to this folder in the new custom field.

Not elegant, but the only solution that comes to my mind right away
